src/software/proto/announcement.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
```

Comment:
since this is used in python_tools, I think it should go in shared/proto

Reply:
moved